iac-extensions/docker/src/main/java/org/sonar/iac/docker/utils/ArgumentUtils.java

```diff
@@ -31,6 +32,17 @@ private ArgumentUtils() {
     // utils class
   }
 
+  /**
+   * Resolve a list of Argument as a single string, used by some instructions like USER.
+   */
+  public static ArgumentResolution resolve(List<Argument> arguments) {
+    StringBuilder sb = new StringBuilder();
+    for (Argument argument : arguments) {
+      sb.append(resolve(argument).value);
+    }
+    return new ArgumentResolution(sb.toString());
+  }
```

Comment:
I don't know if we should merge multiple solutions into a single one. It may make sense in this context, but it does not apply to other situations where it is necessary to handle argument by argument. You could rename the method to something like resolveAndMerge(). This represents what is happening here.